### ISSUE TITLE
No "model" drop down.

### ISSUE THREAD

**Issue by TamedTornado** [cursor/cursor#1049]
Using version 0.14.0, freshly downloaded a few minutes ago.

Despite having an API key, no model dropdown appears. Even doing a workaround found in a previous **closed** issue where you login and logout doesn't cause it to appear. Also, toggling "using key" has no effect.

![image](https://github.com/getcursor/cursor/assets/7510041/59698c95-5064-45d1-9523-51bd6b738e79)

**Comment by jakobFNF** [comment]
The model dropdown is here.

![image](https://github.com/getcursor/cursor/assets/74077124/6064d003-362d-434c-b446-78f2bc2dfd79)

**Comment by mattwebhub** [comment]
@jakobFNF  Ok, but how about the new/other models available?

**Comment by TamedTornado** [comment]
Right, but when I want to create a new project what model does it use for that process? And why can't I set my preference so that I don't have to change that dropdown?
